ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -159,18 +159,19 @@ setupWorkspace() {
 		exit 1;
 	fi
 	
-	mkdir -p "$workFolder"/logFiles
+	mkdir -p ${workFolder}/logFiles
```

Comment:
Not sure what the logic was for removing the quotes here
```suggestion
	mkdir -p "${workFolder}/logFiles"
```

Reply:
I wasn't sure of the logic of having them in the first place